bittensor/_config/__init__.py

```diff
@@ -54,7 +54,7 @@ def __new__( cls, parser: ArgumentParser = None, strict: bool = False, args: Opt
                 Nested config object created from parser arguments.
         """
         if parser == None:
-            parser = ArgumentParser()
+            return config_impl.Config()
```

Comment:
Why is this changed?

Reply:
Because it messes with --help when importing bittensor and using argparse anywhere else.

Reply:
This is because we create an argument parser on bittensor import